### PR TITLE
use pm2 instead of supervisord

### PR DESCRIPTION
**Pull request by daniel-rose**
- add newer php version (7.3)
- refactoring
- makefile for build and push
- add travis workflow
- ...

### CODE REVIEW

File: docker-sync.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+version: '2'
+
+options:
+  verbose: true
+
+syncs:
+  spryker-sync:
+    src: 'PATH_TO_SOURCE'
+    sync_excludes: ['EXCLUDE_1', '...', '.EXCLUDE_N']
```
Comment (sourcecube) [inline docker-sync.yml]: Is that correct?

File: php/7.3/base/apache.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        libzip-dev; \
+    rm -rf /var/lib/apt/lists/*; \
+    \
+    docker-php-ext-configure zip --with-libzip; \
+    docker-php-ext-install zip; \
+    \
+    wget https://raw.githubusercontent.com/composer/getcomposer.org/master/web/installer -O /tmp/installer; \
+    php /tmp/installer --no-ansi --install-dir=/usr/bin --filename=composer --quiet; \
+    rm /tmp/installer; \
+    composer self-update
```
Comment (sourcecube) [inline php/7.3/base/apache.Dockerfile:40]: Is there a reason to add empty lines here?

Comment (daniel-rose) [reply php/7.3/base/apache.Dockerfile:40]: Better to read?!

File: docker-sync.yml (hunk 1 above)
Comment (daniel-rose) [reply docker-sync.yml]: The docker-compose.yml and docker-sync.yml are starting points for beginners.

Comment (daniel-rose) [reply docker-sync.yml]: Maybe the file extension .dist is better for this files.

**Review by sourcecube — APPROVED** [review]

**Review by sourcecube — APPROVED** [review]

**Pull request merged**